### ISSUE TITLE
Error when updating TimeZones?

### ISSUE THREAD

**Issue by ccoffrin** [JuliaTime/TimeZones.jl#279]
This may be a non issue, but thought I would mention just the same.  A recent package update and got the following message,
```
   Building TimeZones → `~/.julia/packages/TimeZones/cAGJs/deps/build.log`
┌ Error: Error building `TimeZones`: 
│ 
│ curl: (22) The requested URL returned error: 503
│ ERROR: LoadError: Unable to automatically install 'tzdata2020a' from '/home/cjc/.julia/packages/TimeZones/cAGJs/Artifacts.toml'
│ Stacktrace:
│  [1] error(::String) at ./error.jl:33
│  [2] ensure_artifact_installed(::String, ::Dict{String,Any}, ::String; platform::Pkg.BinaryPlatforms.Platform, verbose::Bool, quiet_download::Bool) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Artifacts.jl:894
│  [3] do_artifact_str(::String, ::Dict{String,Any}, ::String, ::Module) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Artifacts.jl:1023
│  [4] #invokelatest#1 at ./essentials.jl:712 [inlined]
│  [5] invokelatest at ./essentials.jl:711 [inlined]
│  [6] macro expansion at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Artifacts.jl:1058 [inlined]
│  [7] build(::String, ::Array{String,1}, ::String, ::String, ::String; verbose::Bool) at /home/cjc/.julia/packages/TimeZones/cAGJs/src/tzdata/build.jl:52
│  [8] build(::String) at /home/cjc/.julia/packages/TimeZones/cAGJs/src/tzdata/build.jl:113
│  [9] build(::String; force::Bool) at /home/cjc/.julia/packages/TimeZones/cAGJs/src/build.jl:11
│  [10] build at /home/cjc/.julia/packages/TimeZones/cAGJs/src/build.jl:11 [inlined] (repeats 2 times)
│  [11] top-level scope at /home/cjc/.julia/packages/TimeZones/cAGJs/deps/build.jl:3
│  [12] include(::String) at ./client.jl:439
│  [13] top-level scope at none:5
│ in expression starting at /home/cjc/.julia/packages/TimeZones/cAGJs/deps/build.jl:3

└ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Operations.jl:899
```

However another call to `] build TimeZones` seemed to work fine.

**Comment by omus** [comment]
Thanks for the report. We can possibly implement an automated retry but I'd like to see how rare this issue is first

**Comment by ccoffrin** [comment]
sounds good.

**Comment by oxinabox** [comment]
This error occurred today for one of our internal packages during nightly CI.

**Comment by omus** [comment]
In PR https://github.com/JuliaTime/TimeZones.jl/pull/441 the `deps/build.jl` was removed so this is no longer an issue. Included in TimeZones release 1.12
